crates/net/network/src/config.rs

```diff
@@ -253,20 +259,34 @@ impl<C> NetworkConfigBuilder<C> {
         self
     }
 
-    /// Sets the discv4 config to use.
+    /// Sets the boot nodes.
     pub fn boot_nodes(mut self, nodes: impl IntoIterator<Item = NodeRecord>) -> Self {
         self.boot_nodes = nodes.into_iter().collect();
         self
     }
 
+    /// Sets the trusted nodes.
+    pub fn trusted_nodes(mut self, nodes: impl IntoIterator<Item = NodeRecord>) -> Self {
+        self.trusted_nodes = nodes.into_iter().collect();
+        self
+    }
```

Comment:
same, +1 `add_` function

Reply:
Renamed to `add_trusted_nodes` and added `add_trusted_node`